src/utilities/fillIn.ts

```diff
@@ -95,7 +95,7 @@ export function fillInFieldConfig(fieldConfig: IFieldConfigPartial): IFieldConfi
   const requiredValidationRule: undefined | { [key: string]: Rule } = fieldConfig.required
     ? {
         required: {
-          message: `Required - Please input a valid ${label || 'value'}`,
+          message: ` ${`'${label}'` || 'This field'} is required`,
```

Comment:
I tried running this code with `label` as `undefined`, and the message printed is `'undefined' is required`. I'm assuming adding the quotation marks around the value of `label` causes the value to no longer be falsy.

Reply:
YUP, turns out label will always be defined, so I rethought the expected result and wrote tests.